CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Change Log
 
+## [1.2.2](https://github.com/ably/ably-cocoa/tree/1.2.2)
+
+[Full Changelog](https://github.com/ably/ably-cocoa/compare/1.2.1...1.2.2)
+
+**Highlights**
+
+***Supports Build with Xcode 12***
+
+Fixes the Carthage build. Increases the minimum deployment target for iOS, from 8 to 9 (conflated in [\#1075](https://github.com/ably/ably-cocoa/pull/1075)).
+
+***New Push Notification Registration API for apps using the SwiftUI App Lifecycle***
+
+For new Swift-based apps created with Xcode 12, the default lifecycle is SwiftUI App.
+This means that the previous approach we supported of conforming to `ARTPushRegistererDelegate` on your `UIApplicationDelegate` no longer works.
+
+Going forward apps should use the `pushRegistererDelegate` property on client options to supply this delegate ([\#1075](https://github.com/ably/ably-cocoa/pull/1075)).
+
+***Reduced Dependencies***
+
+We've removed both KSCrash ([\#1073](https://github.com/ably/ably-cocoa/pull/1073)) and SAMKeychain ([\#1067](https://github.com/ably/ably-cocoa/pull/1067)).
+
+KSCrash, in particular, presented a heavy build burden and was causing issues for some users at runtime in respect of how it was catching unhandled exceptions and reporting them to Ably's Sentry service.
+The strategy for handling uncaught exceptions, including how to log or transmit them, should be selected and implement by the app developer, not a library dependency.
```

Comment:
In Cocoa there is no effective way to catch selectively only those exceptions that occur within the library, and the attempt to do that using KCrash was interfering with app developers' own exception handling. Our advice is that app developers incorporate their own application-wide exception handling and crash reporting.

Reply:
https://github.com/ably/ably-cocoa/pull/1081/commits/38398b1e2ac338294174e890ca2ef0a21bcd1076